install_pim/manual/system_requirements/system_requirements.rst.inc

```diff
@@ -3,9 +3,9 @@
 Akeneo PIM only works on Linux servers. The application’s behaviour has been tested on several operating systems. We cannot guarantee the behaviour of the application on servers different from:
 
 +------------------------+-------------------+
-| Debian (Linux)         | ≥ 9 (64 bits)     |
+| Debian (Linux)         | ≥ 10 (64 bits)    |
 +------------------------+-------------------+
-| Ubuntu (Linux)         | ≥ 16.04 (64 bits) |
+| Ubuntu (Linux)         | ≥ 18.04 (64 bits) |
 +------------------------+-------------------+
 | CentOS / RHEL* (Linux) | ≥ 7               |
```

Comment:
should we update to RHEL 8?

Reply:
Good question... That means we should test on it... Can you do a card for that? Thanks!